search/api.py

```diff
@@ -0,0 +1,261 @@
+"""
+Functions for search
+"""
+from itertools import islice
+import logging
```

Comment:
More like a general convention 
http://stackoverflow.com/a/20763446/257092
But feel free to leave it as is